frappe/www/rss.py

```diff
@@ -17,16 +17,15 @@ def get_context(context):
 
 	blog_list = frappe.get_all(
 		"Blog Post",
-		fields=["name", "published_on", "modified", "title", "content"],
+		fields=["name", "published_on", "modified", "title", "blog_intro", "route"],
 		filters={"published": 1},
 		order_by="published_on desc",
 		limit=20,
 	)
 
 	for blog in blog_list:
-		blog_page = cstr(quote(blog.name.encode("utf-8")))
-		blog.link = urljoin(host, blog_page)
-		blog.content = escape_html(blog.content or "")
+		blog.link = urljoin(host, blog.route)
+		blog.blog_intro = escape_html(blog.blog_intro or "")
```

Comment:
Unrelated. 

This shouldn't be escaped ideally, html2text makes more sense here?

Reply:
Ah `escape_html` is actually exactly the right thing:

```python
def escape_html(text: str) -> str:
	if not isinstance(text, str):
		return text

	html_escape_table = {
		"&": "&amp;",
		'"': "&quot;",
		"'": "&apos;",
		">": "&gt;",
		"<": "&lt;",
	}

	return "".join(html_escape_table.get(c, c) for c in text)
```

Reply:
For intro it's fine, content can be HTML too.